config/src/config/consensus_config.rs

```diff
@@ -32,7 +33,8 @@ pub struct ConsensusConfig {
 impl Default for ConsensusConfig {
     fn default() -> ConsensusConfig {
         ConsensusConfig {
-            max_block_size: 6000,
+            max_block_txns: 6000,
+            max_block_bytes: 5 * 1024 * 1024, // 5MB
```

Comment:
This number looks small to me. How did you get this number? Do you have a rough estimate of how many transactions on average we can pack with this? Ideally this should allow 4-5k average-sized txns?